src/EntityFrameworkCore.Testing.Moq/Extensions/DbContextExtensions.cs

```diff
@@ -139,10 +140,12 @@ public static TDbContext AddExecuteSqlRawResult<TDbContext>(
                 .RawSqlCommandBuilder;
 
             Mock.Get(existingRawSqlCommandBuilder)
-                .Setup(m => m.Build(It.Is<string>(s => s.Contains(sql, StringComparison.OrdinalIgnoreCase)),
-                    It.Is<IEnumerable<object>>(p => ParameterMatchingHelper.DoInvocationParametersMatchSetUpParameters(parameters, p))))
-                .Returns((string providedSql, IEnumerable<object> providedParameters) => rawSqlCommand)
-                .Callback((string providedSql, IEnumerable<object> providedParameters) =>
+                .Setup(m => m.Build(
+                    It.Is<string>(s => s.Contains(sql, StringComparison.OrdinalIgnoreCase)),
+                    It.Is<IEnumerable<object>>(p => ParameterMatchingHelper.DoInvocationParametersMatchSetUpParameters(parameters, p)),
+                    It.IsAny<IModel>()))
+                .Returns((string _, IEnumerable<object> _, IModel _) => rawSqlCommand)
+                .Callback((string providedSql, IEnumerable<object> providedParameters, IModel _) =>
                 {
                     callback?.Invoke(providedSql, providedParameters);
```

Comment:
Can't recall why the `callback?.Invoke(providedSql, providedParameters);` is used, but a chance we may need to pass the model
I'll catch this on a subsequent branch

Reply:
Checked this, looks like a feature I added a while ago, no action required